css/style-wider-screen.css

```diff
@@ -0,0 +1,343 @@
+/* #site-wrapper {
+    display: flex;
+    flex-direction: column;
+} */
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+#header-background {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  background-image: url("../img/Header\ bg.svg");
+  background-size: 100%;
+  background-repeat: no-repeat;
+  flex-direction: column;
+  order: 0;
+}
+
+#logo {
+  padding: 0 120px;
+  display: flex;
+  align-items: center;
+}
+
+#logo a {
+  width: 76px;
+  height: 20px;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  justify-content: center;
+}
+
+/* #flex-nav {
+    display: flex;
+    flex-direction: row;
+    gap: 10em;
+} */
+
+.burguer-menu-vector {
+  display: none !important;
+}
+
+#burguer-menu-icon {
+  display: flex;
+  flex-direction: row;
+}
+
+.wide-screen-nav {
+  list-style: none;
+  display: flex !important;
+  flex-direction: row;
+  align-items: center;
+  gap: 8px;
+  width: 259px;
+  height: 40px;
+  margin-right: 5px;
+  padding: 0 120px;
+}
+
+.wide-screen-nav li {
+  font-size: 15px;
+  line-height: 20px;
+  color: #344563;
+}
+
+#primary-text {
+  width: 60%;
+  height: 268px;
+  top: calc(50% - 268px / 2);
+}
+
+#hello-text {
+  font-size: 20px;
+  line-height: 28px;
+}
+
+#works {
+  width: 100%;
+  height: 2694px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding-top: 0;
+  margin-top: 0;
+  gap: 142px;
+}
+
+.works-item {
+  display: flex;
+  flex-direction: row;
+  width: 80%;
+  height: 496px;
+  border-radius: 24px;
+}
+
+.works-item-image {
+  width: 544px;
+  height: 448px;
+  flex: 1 1;
+  margin: 0 40px;
+}
+
+.works-item-info-block {
+  flex: 1 1;
+}
+
+.works-item-image-1 {
+  background: url("../img/Snapshoot\ Portfolio\ wide\ screen.png");
+}
+
+.works-item-image-2 {
+  background: url("../img/Snapshoot\ Portfolio\ wide\ screen\ 2.png");
+}
+
+.works-item-image-3 {
+  background: url("../img/Snapshoot\ Portfolio\ wide\ screen\ 3.png");
+}
+
+.works-item-image-4 {
+  background: url("../img/Snapshoot\ Portfolio\ wide\ screen\ 4.png");
+}
+
+.works-item-title {
+  width: 496px;
+  height: 46px;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #091e42;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.works-item-project-info-canopy {
+  width: 75px;
+  height: 24px;
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 24px;
+
+  /* identical to box height, or 133% */
+
+  /* Theme 1 / Neutrals 3 */
+
+  color: #344563;
+
+  /* Inside auto layout */
+
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.works-item-project-description {
+  width: 100%;
+  height: 50px;
+}
+
+#about {
+  justify-content: center;
+  height: 724px;
+  left: 1440px;
+  background: #fff;
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+}
+
```

Comment:
[OPTIONAL] At screen size between 768px and 1150px, part of the about section pushes to the left side and goes missing. This does not make your project smooth and responsive. So kindly fix that. :pray: 


**At screen width 816px**

![Screenshot from 2022-09-30 16-52-04](https://user-images.githubusercontent.com/65233860/193319523-5ed11573-95df-47fa-9e4e-c0f959153b6f.png)

**At screen width 1074px**

![Screenshot from 2022-09-30 16-50-28](https://user-images.githubusercontent.com/65233860/193319551-3e1ecd2f-f85c-4dc0-b83e-011d7bf8cda1.png)